test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/ObjectManagement/DatabaseHandlerTests.cs

```diff
@@ -162,6 +166,111 @@ public async Task DatabaseAlreadyExistsErrorTest()
             }
         }
 
+        [Test]
+        public void GetAzureEditionsTest()
+        {
+            var actualEditions = DatabaseHandler.AzureEditions;
+            var expectedEditions = new string[]
+            {
+                AzureEdition.Basic.DisplayName,
+                AzureEdition.Standard.DisplayName,
+                AzureEdition.Premium.DisplayName,
+                AzureEdition.DataWarehouse.DisplayName,
+                AzureEdition.BusinessCritical.DisplayName,
+                AzureEdition.GeneralPurpose.DisplayName,
+                AzureEdition.Hyperscale.DisplayName
+            };
+            Assert.That(actualEditions, Is.EquivalentTo(expectedEditions));
```

Comment:
What exactly is the point of this test? Seems like to actually be useful it should be verifying that all the editions listed in AzureEdition are returned by AzureEditions. Right now if I add a new edition to AzureEdition but forget to update that helper method then this test won't catch that...

Same question applies to the other similar tests below.